README.rst

```diff
@@ -279,6 +279,25 @@ colors`_ are supported too:
 
       * <fg=blue>Read</> the documentation at <comment>https://symfony.com/doc</>
 
+Validation
+----------
+
+When submitting a recipe, several checks are automatically executed to validate
+the recipe:
+
+* YAML files suffix must be `.yaml`, not `.yml`;
+* YAML files must be valid;
+* JSON files must be valid;
+* Aliases are only supported in the main repository, not the contrib one;
+* Aliases must not be already defined by another package;
+* The versions are valid;
```

Comment:
"The versions are valid" -> "All the versions listed in the manifest exist as package versions on Packagist" ?